cmd/admin-handlers.go

```diff
@@ -1768,6 +1769,36 @@ func getServerInfo(ctx context.Context, r *http.Request) madmin.InfoMessage {
 	}
 }
 
+func getKubernetesInfo() madmin.KubernetesInfo {
+	ctx, cancel := context.WithTimeout(context.Background(), 10*time.Second)
+	defer cancel()
```

Comment:
Accept a context from caller and let them set the timeout.

Reply:
Done